x-pack/plugins/cloud_security_posture/server/create_transforms/create_transforms.ts

```diff
@@ -9,11 +9,18 @@ import { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/typ
 import type { ElasticsearchClient, Logger } from '@kbn/core/server';
 import { errors } from '@elastic/elasticsearch';
 import { latestFindingsTransform } from './latest_findings_transform';
+import { latestVulnerabilitiesTransform } from './latest_vulnerabilities_transforms';
 
 const LATEST_TRANSFORM_V830 = 'cloud_security_posture.findings_latest-default-0.0.1';
 const LATEST_TRANSFORM_V840 = 'cloud_security_posture.findings_latest-default-8.4.0';
-
-const PREVIOUS_TRANSFORMS = [LATEST_TRANSFORM_V830, LATEST_TRANSFORM_V840];
+const LATEST_TRANSFORM_V880_001 = 'cloud_security_posture.findings_latest-default-0.0.1';
+const LATEST_VULNERABILITIES_V880 = 'cloud_security_posture.vulnerabilities_latest-default-8.8.0';
+const PREVIOUS_TRANSFORMS = [
+  LATEST_TRANSFORM_V830,
+  LATEST_TRANSFORM_V840,
+  LATEST_TRANSFORM_V880_001,
+  LATEST_VULNERABILITIES_V880,
+];
```

Comment:
`PREVIOUS_TRANSFORMS` array used to delete old Transform version. 
Since you are just creating the first one, what  the purpose of `LATEST_TRANSFORM_V880_001` and `LATEST_VULNERABILITIES_V880`?

Reply:
Looking after this 
`
const LATEST_TRANSFORM_V830 = 'cloud_security_posture.findings_latest-default-0.0.1';
const LATEST_TRANSFORM_V840 = 'cloud_security_posture.findings_latest-default-8.4.0';
`LATEST_TRANSFORM_V880_001 will be the first version and the `LATEST_VULNERABILITIES_V880` release version. 

What does the difference between  `LATEST_TRANSFORM_V830` and  `LATEST_TRANSFORM_V840`?
 @CohenIdo

Reply:
Hey @Omolola-Akinleye, you can find the changes between V8.30 and V8.40 [here](https://github.com/elastic/kibana/pull/136766/files#diff-99cdc301f7656a8608fc7a73f7998ca59d327d71ba9f6c4ba1d0a89838f9f264R19).

For a better understanding of the logic behind this code, consider this scenario: in a month after the vul transform is included in 8.8, we will want to change the Transform logic / we found a bug. 
Whenever a user is upgrading from 8.8 to 8.9, we want to remove the old Transform and replace it with the new one. 
Our goal is to provide frictionless upgrades, which is why we keep the IDs of previous Transforms